grpcbp/tracing.go

```diff
@@ -0,0 +1,40 @@
+package grpcbp
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/reddit/baseplate.go/tracing"
+	"google.golang.org/grpc/metadata"
+)
+
+const ()
+
+func CreateGRPCContextFromSpan(ctx context.Context, span *tracing.Span) context.Context {
+	ctx = metadata.AppendToOutgoingContext(ctx,
+		HeaderTracingTrace, span.TraceID(),
+		HeaderTracingSpan, span.ID(),
+		HeaderTracingFlags, strconv.FormatInt(span.Flags(), 10),
+	)
+	if span.ParentID() != "" {
+		ctx = metadata.AppendToOutgoingContext(ctx,
+			HeaderTracingParent, span.ParentID(),
+		)
+	} else {
+		md, _ := metadata.FromIncomingContext(ctx)
+		delete(md, HeaderTracingParent)
```

Comment:
same key canonicalization issue, that direct `delete` probably won't work.

looking at the doc, I think the way to unset a header is to explicitly set `""` as the value:
```
md.Set(HeaderTracingParent, "")
```